entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityOperationHandler.kt

```diff
@@ -163,6 +167,67 @@ class EntityOperationHandler(
             ResponseEntity.status(HttpStatus.MULTI_STATUS).body(batchOperationResult)
     }
 
+    /**
+     * Implements 6.16.3.1 - Update Batch of Entities
+     */
+    @PostMapping("/update", consumes = [MediaType.APPLICATION_JSON_VALUE, JSON_LD_CONTENT_TYPE])
+    suspend fun update(
+        @RequestHeader httpHeaders: HttpHeaders,
+        @RequestBody requestBody: Mono<String>,
+        @RequestParam options: Optional<String>
+    ): ResponseEntity<*> {
+        val userId = extractSubjectOrEmpty().awaitFirst()
+        val body = requestBody.awaitFirst()
+        checkContext(httpHeaders, body)
+        val context = getContextFromLinkHeader(httpHeaders.getOrEmpty(HttpHeaders.LINK))
+        val disallowOverwrite = options.map { it == "noOverwrite" }.orElse(false)
+
+        val (extractedEntities, jsonLdEntities, ngsiLdEntities) =
+            extractAndParseBatchOfEntities(body, context, httpHeaders.contentType)
+        val (existingEntities, newEntities) = entityOperationService.splitEntitiesByExistence(ngsiLdEntities)
+
+        val updateBatchOperationResult = when {
+            newEntities.isEmpty() -> BatchOperationResult()
+            authorizationService.userCanUpdateEntity(userId.toUri(),userId) -> entityOperationService.update(newEntities)
+            else -> BatchOperationResult(
+                errors = ArrayList(
+                    newEntities.map { BatchEntityError(it.id, arrayListOf("User forbidden to create entities")) }
+                )
+            )
+        }
+        val existingEntitiesIdsAuthorized =
```

Comment:
the authorization check here is more or less the same than the one done in L191.